style.css

```diff
@@ -0,0 +1,122 @@
+body {
+    background: #7faeff;
+}
+
+h1 {
+    font-size: 35px;
+    text-align: center;
+    background: #efeec3;
+    margin: 40px; 
+}
+
+main {
+    font-family: Arial, Helvetica, sans-serif;
+    line-height: 1.8;
+    width: 75%; 
+    margin: 0 auto;
+    background: #e4f9f6;
+}
+
+section {
+    display: flex;
+    width: 100%;
+}
+
+p {
+    font-weight: bold;
+    background: #efeec3;
+    margin: 20px 0 0 0;
+}
+
+.codexample {
+    background: #0000FF; 
+}
+
+.codexampl {
+    font-size: 130%;
+    color: #fff;
+    white-space: pre-wrap; 
+    display: flex;
+    flex-direction: column; 
+}
+
+code {
+    justify-content: space-between;
+}
+
+ul {
+    list-style-type: none;
+}
+
+li:nth-child(1) {
```

Comment:
Would be better to add class to elements. It makes code more readable and easy to reuse